src/conduits/RwaInputConduit3.sol

```diff
@@ -267,7 +278,13 @@ contract RwaInputConduit3 {
                     Math
     //////////////////////////////////*/
 
+    uint256 constant WAD = 10**18;
```

Comment:
```suggestion
    uint256 internal constant WAD = 10**18;
```

Reply:
There's a similar issue in the output conduit.

Reply:
Done